packages/quantic/README.md

```diff
@@ -24,9 +24,9 @@
 
 ### Install VSCode Extension
 
-[vscode Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)
+Optionally install the [VSCode Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) if you do not want use the equivalent `sfdx-cli` commands to setup.
```

Comment:
Unless there are quantic dev workflows other than setup that use VSCode without (specifying) alternate sfdx commands.